src/hermezjs-typings.d.ts

```diff
@@ -388,6 +393,10 @@ declare module "@hermeznetwork/hermezjs/src/tx" {
     nonce: number;
   }
 
+  interface TxData {
```

Comment:
This interface should be now removed since after the merge it's duplicated.

Reply:
Done!